common/utils/src/main/scala/org/apache/spark/SparkThrowableHelper.scala

```diff
@@ -85,20 +85,20 @@ private[spark] object SparkThrowableHelper {
         toJsonString { generator =>
           val g = generator.useDefaultPrettyPrinter()
           g.writeStartObject()
-          g.writeStringField("errorClass", "LEGACY")
+          g.writeStringField("condition", "LEGACY")
```

Comment:
How about `errorCondition`? The name condition makes confusion for developers who not read the document common/utils/src/main/resources/error/README.md.